views/pages/add-user.php

```diff
@@ -0,0 +1,78 @@
+
+<?php
+
+require_once '../../config/dbauth.php';
+
+$conn = new mysqli($hn, $un, $pw, $db);
+if($conn->connect_error) die($conn->connect_error);
+
+
+
+function add_user($conn, $AccountID, $FirstName, $LastName, $Phone,  $Email, $Street, $City, $State, $Zip,  $StartDate, $token, $AccountTypeID){
+	//code to add user here
+	$query = "INSERT INTO LIB_ACCOUNT (AccountID, FirstName, LastName, Phone, Email, Street, City, State, Zip, StartDate, Password, AccountTypeID) VALUES ('$AccountID', '$FirstName', '$LastName', '$Phone', '$Email', '$Street', '$City', '$State', '$Zip', '$StartDate', '$token', '$AccountTypeID')";
+	$result = $conn->query($query);
+	if(!$result) die($conn->error);
+}
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+	$AccountID = $_POST['AccountID'];
+	$FirstName = $_POST['FirstName'];
+	$LastName = $_POST['LastName'];
+	$Phone = $_POST['Phone'];
+	$Email = $_POST['Email'];
+	$Street = $_POST['Street'];
+	$City = $_POST['City'];
+	$State = $_POST['State'];
+	$Zip = $_POST['Zip'];
+	$StartDate = $_POST['StartDate'];
+	$Password = $_POST['Password'];
+	$AccountTypeID = $_POST['AccountTypeID'];
+
+	
+	$token = password_hash($Password,PASSWORD_DEFAULT); 
+
+	add_user($conn, $AccountID, $FirstName, $LastName, $Phone,  $Email, $Street, $City, $State, $Zip,  $StartDate, $token, $AccountTypeID);
+	
+		
+
+	
+header("Location: home.php");
+exit();
+
+
+
+
+}
+?>
+
+<html>
+	<head>
+	
+	</head>
+	
+	<body>
+		<form method='post' action='add-user.php'>
+			<pre>
+				AccountID: <input type='text' name='AccountID'>
+				First Name: <input type='text' name='FirstName'>
+				Last Name: <input type='text' name='LastName'>
+				Phone: <input type='text' name='Phone'>
+				Email: <input type='text' name='Email'>
+				Street: <input type='text' name='Street'>
+				City: <input type='text' name='City'>
+				State: <input type='text' name='State'>
+				Zip: <input type='text' name='Zip'>
+				Start Date: <input type='date_create' name='StartDate'>
+				Password: <input type='password' name='Password'>
+				Account Type: <select name='AccountTypeID'>
```

Comment:
This field should also be removed from the form. No user registering for an account should be able to determine their own account type. Instead, our SQL should just default them to be  'Member'.